src/hand-tracking/src/iCubCamera.cpp

```diff
@@ -144,7 +145,7 @@ bool iCubCamera::bufferData()
     if (tmp_imgin != YARP_NULLPTR)
     {
         init_img_in_ = true;
-        image_ = cv::cvarrToMat(tmp_imgin->getIplImage()).clone();
+        image_ = yarp::cv::toCvMat(*tmp_imgin).clone();
```

Comment:
We should remember to enforce the specific version of YARP becuase of this change.

Reply:
The version associated to it (https://github.com/robotology/yarp/commit/422b27a75a2f18fc08de9e216c3dbd52f5a22909#diff-8923ab12f8f8950cc1ffd26067e20f38) seems to be `3.1.100`

Reply:
Ok fine 👍